test/validation/api/hash/hash.c

```diff
@@ -238,6 +240,85 @@ static void hash_test_crc32(void)
 	}
 }
 
+static void hash_test_ethernet_crc32(void)
+{
+	uint32_t ret;
+	uint32_t verify = 0xdebb20e3;
+
+	ret = odp_hash_crc32(test_packet_ipv4_udp_64_crc,
+			     sizeof(test_packet_ipv4_udp_64_crc), CRC32_INIT);
+	CU_ASSERT(ret == verify);
+
+	ret = odp_hash_crc32(test_packet_ipv4_udp_68_crc,
+			     sizeof(test_packet_ipv4_udp_68_crc), CRC32_INIT);
+	CU_ASSERT(ret == verify);
+
+	ret = odp_hash_crc32(test_packet_ipv4_udp_70_crc,
+			     sizeof(test_packet_ipv4_udp_70_crc), CRC32_INIT);
+	CU_ASSERT(ret == verify);
+
+	ret = odp_hash_crc32(test_packet_ipv4_udp_71_crc,
+			     sizeof(test_packet_ipv4_udp_71_crc), CRC32_INIT);
+	CU_ASSERT(ret == verify);
+
+	ret = odp_hash_crc32(test_packet_ipv4_udp_287_crc,
+			     sizeof(test_packet_ipv4_udp_287_crc), CRC32_INIT);
+	CU_ASSERT(ret == verify);
+
+	ret = odp_hash_crc32(test_packet_ipv4_udp_400_crc,
+			     sizeof(test_packet_ipv4_udp_400_crc), CRC32_INIT);
+	CU_ASSERT(ret == verify);
+
+	ret = odp_hash_crc32(test_packet_ipv4_udp_503_crc,
+			     sizeof(test_packet_ipv4_udp_503_crc), CRC32_INIT);
+	CU_ASSERT(ret == verify);
+}
+
+static void hash_test_ethernet_crc32_odd_align(void)
+{
+	uint32_t ret, size;
+	uint32_t verify = 0xdebb20e3;
```

Comment:
This could be defined in just one place. Also, could be const.

Reply:
Fixed in v2. Changed it to a define.